hive/main.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+from pprint import pprint
+from hive.runners.utils import load_config
+from hive.runners import get_runner
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-c", "--config")
+    parser.add_argument("-p", "--preset-config")
```

Comment:
It would be nice to add some descriptions for each one of the arguments. What is `preset-config` for example?